napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -105,17 +110,16 @@ def set_curr_img(self, curr_img: Optional[Dict[str, str]] = None):
         if curr_img is not None:
             self.curr_img = curr_img
             path: str = curr_img["File Path"]
-            if path not in self.annotation_dict.keys():
-                self.annotation_dict.update(
-                    {path: [curr_img["File Name"], curr_img["FMS"]]}
-                )
+            if len(self.annotation_dict[path]) < 3:
+
                 self.view.render_default_values()
             else:
                 self.view.render_values(self.annotation_dict[path][2::])
-            self.view.set_curr_index(curr_img["Row"])
+            self.view.set_curr_index(int(curr_img["Row"]))
```

Comment:
what does curr_img store?

Reply:
Putting a bunch of comments on this method . curr_img = {'File Path' : 'path', 'Row' : str(rownum)}